MathlibTest/TransImports.lean

```diff
@@ -0,0 +1,10 @@
+import Mathlib.Util.TransImports
+
+/--
+info: 'MathlibTest.TransImports' has 483 transitive imports
```

Comment:
Having the tests fail when the actual *number* of transitive imports changes may be a little too volatile.

Reply:
Yeah ... should we leave out the test? Or (add an option to) remove the total count from the output?

Reply:
I decided to add a second optional argument to the command: `#trans_imports "Lean.Elab.I" at_most 500` confirms that the number of transitive imports is at most 500, but does not print the actual number.  This should be more stable and might even be useful to limit the import size of certain files.

Reply:
Thanks!
maintainer merge